tests/v2/test_waku_noise.nim

```diff
@@ -0,0 +1,82 @@
+{.used.}
+
+import
+  testutils/unittests,
+  ../../waku/v2/protocol/waku_noise/noise,
+  ../test_helpers,
+  std/tables
+
+procSuite "Waku Noise":
+  
+  let rng = rng()
+
+  test "Encrypt -> decrypt public keys":
+
+    let noisePublicKey: NoisePublicKey = genNoisePublicKey(rng[])
+
+    let 
+      cs: ChaChaPolyCipherState = randomChaChaPolyCipherState(rng[])
+      enc_pk: NoisePublicKey = encryptNoisePublicKey(cs, noisePublicKey)
+      dec_pk: NoisePublicKey = decryptNoisePublicKey(cs, enc_pk)
+
+    check: 
+      noisePublicKey == dec_pk
+
+  test "Decrypt unencrypted public key":
+
+    let noisePublicKey: NoisePublicKey = genNoisePublicKey(rng[])
+
+    let 
+      cs: ChaChaPolyCipherState = randomChaChaPolyCipherState(rng[])
+      dec_pk: NoisePublicKey = decryptNoisePublicKey(cs, noisePublicKey)
+
+    check:
+      noisePublicKey == dec_pk
+
+  test "Encrypt -> encrypt public keys":
+
+    let noisePublicKey: NoisePublicKey = genNoisePublicKey(rng[])
+
+    let
+      cs: ChaChaPolyCipherState = randomChaChaPolyCipherState(rng[])
+      enc_pk: NoisePublicKey = encryptNoisePublicKey(cs, noisePublicKey)
+      enc2_pk: NoisePublicKey = encryptNoisePublicKey(cs, enc_pk)
+    
+    check enc_pk == enc2_pk
```

Comment:
```suggestion
    check:
      enc_pk == enc2_pk
```

For consistency with other tests.

Reply:
Addressed in https://github.com/status-im/nim-waku/commit/704f3fe40fb4ad63212ec5f6face687b4971c909